### PR TITLE
619 Inform user that the local db is out of sync

### LINKED ISSUE #619: Show error message to user if jore-map-ui DB was out of sync with Jore DB (when updating node or routePath)
* Update function in our DB throws 409 error code. If frontend receives 409, show error "Karttakäyttöliittymän sisäinen tietokanta oli epäsynkassa jore tietokannan kanssa. Päivitä sivu ja yritä uudelleen."

### PR DESCRIPTION
**Pull request by kronken**
Closes #619 

Test by:
- Make some item out of sync
(Easiest way is to comment node syncing in BE, and then update a nodes name. This save will be successful, but will not be synced to local DB, resulting in a confict)
- Save the item

### CODE REVIEW

File: src/components/sidebar/routePathView/RoutePathView.tsx [hunk 1]
```diff
@@ -173,8 +174,7 @@ class RoutePathView extends ViewFormBase<IRoutePathViewProps, IRoutePathViewStat
 
             DialogStore.setFadeMessage('Tallennettu!');
         } catch (err) {
-            const errMessage = err.message ? `, (${err.message})` : '';
-            this.props.errorStore!.addError(`Tallennus epäonnistui${errMessage}`);
+            showError(err);
```
Comment (Divinare) [inline src/components/sidebar/routePathView/RoutePathView.tsx]: showError is too generic name.

File: src/components/sidebar/networkView/nodeView/NodeView.tsx [hunk 2]
```diff
@@ -98,8 +99,7 @@ class NodeView extends ViewFormBase<INodeViewProps, INodeViewState> {
             this.props.nodeStore!.setCurrentStateAsOld();
             this.props.dialogStore!.setFadeMessage('Tallennettu!');
         } catch (err) {
-            const errMessage = err.message ? `, (${err.message})` : '';
-            this.props.errorStore!.addError(`Tallennus epäonnistui${errMessage}`);
+            showError(err);
```
Comment (Divinare) [inline src/components/sidebar/networkView/nodeView/NodeView.tsx]: What was wrong with how it was before?

Comment (Divinare) [reply src/components/sidebar/networkView/nodeView/NodeView.tsx]: As in #662, I would add error param to addMessage

```
this.props.errorStore!.addError(Tallennus epäonnistui., error);

@action
public addMessage(message: string, error?: Error) {
    let msg = message;
    const statusCode = error.errorCode;
    if(error.errorCode) {
           msg += ' ' + httpStatusCodeList[error.errorCode];
     }
    this._errors.push(msg);
    /* tslint:disable no-console */
    if(error) console.error(error);
    /* tslint:enable no-console */
}
```

Comment (kronken) [reply src/components/sidebar/networkView/nodeView/NodeView.tsx]: Refactor to using logic more similar to this example

**Pull request merged**